runtime/libffi/preconf/xz64/ffi.h

```diff
@@ -96,7 +96,7 @@ extern "C" {
 #  endif
 #  ifdef _AIX /* AIX 5.1 and earlier have LONGLONG_MAX */
 #   ifndef __PPC64__
-#    if defined (__IBMC__) || defined (__IBMCPP__)
+#    if defined (__IBMC__) || defined (__IBMCPP__) || defined(__open_xl__)
```

Comment:
for the above dozen ffi.h files on non-AIX platforms, do we really need to change this line of code, especially since it is under` #  ifdef _AIX ` to begin with.  in other words, how changing or not-changing this code affect the builds on non-AIX platforms?